source/includes/harvest/_scheduled_interviews.md

```diff
@@ -303,3 +326,235 @@ Retrieve an interview by its ID.
 Parameter | Description
 --------- | -----------
 id | The ID of the scheduled interview to retrieve
+
+## Post: Create Scheduled Interview
+
+```shell
+curl -X POST 'https://harvest.greenhouse.io/v1/scheduled_interviews/{id}'
```

Comment:
What ID do you have to include here? It looks like all the relevant information is in the body, right?

Reply:
yeah it should be obvious that `:id` here refers to the Scheduled Interview ID - we used this syntax when we describe getting a scheduled interview.

Reply:
But this is a POST, so there isn't an ID to reference, right?

Reply:
ooooohhh snap yup thats a typo good catch